src/button/template/componentTemplate.jsx

```diff
@@ -403,8 +403,8 @@ function renderTagline({ label, tagline, color, locale, multiple, env, cards, ch
         return;
     }
   
-    
-    if (__WEB__ && layout !== BUTTON_LAYOUT.VERTICAL) {
+    // tagline is only supported in horizontal layout
+    if (layout === BUTTON_LAYOUT.VERTICAL) {
```

Comment:
We also need to `return;` if `__WEB__` is true. We only show tagline on second render.

Reply:
Updating...